ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -308,6 +308,15 @@ aws|aws-arm64|aws-usgov)
       inject_spot_instance_config ${dir}
     fi
     ;;
+  gcp)
+    if [ X"${WORKER_RHCOS_IMAGE_FROM_GCP_MARKETPLACE}" != X"" ]; then
+      echo "Using GCP marketplace RHCOS image '${WORKER_RHCOS_IMAGE_FROM_GCP_MARKETPLACE}' for compute machinesets..."
+      rhcos_image="${WORKER_RHCOS_IMAGE_FROM_GCP_MARKETPLACE//\//+}"
+      sed -i -r "s/image: .+$/image: ${rhcos_image}/" "${dir}"/openshift/*worker-machineset*.yaml
```

Comment:
@jianli-wei maybe we can try to use another separator for sed command to avoid dealing with the "/" character.
such as "#"

```
[root@preserve-gpei-worker openshift]# echo $WORKER_RHCOS_IMAGE_FROM_GCP_MARKETPLACE
projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-48-x86-64-202210040145

[root@preserve-gpei-worker openshift]# grep image 99_openshift-cluster-api_worker-machineset-3.yaml
            image: projects/rhcos-cloud/global/images/rhcos-412-86-202209302317-0-gcp-x86-64

[root@preserve-gpei-worker openshift]# sed -i "s#image: .*#image: ${WORKER_RHCOS_IMAGE_FROM_GCP_MARKETPLACE}#" 99_openshift-cluster-api_worker-machineset-3.yaml

[root@preserve-gpei-worker openshift]# grep image 99_openshift-cluster-api_worker-machineset-3.yaml
            image: projects/redhat-marketplace-public/global/images/redhat-coreos-ocp-48-x86-64-202210040145
```

Reply:
@gpei got it, thanks!